akka-remote/src/main/scala/akka/remote/artery/Control.scala

```diff
@@ -29,22 +29,22 @@ private[akka] trait Reply extends ControlMessage
  * Marker trait for control messages that can be sent via the system message sub-channel
  * but don't need full reliable delivery. E.g. `HandshakeReq` and `Reply`.
  */
-private[akka] trait ControlMessage
+private[akka] trait ControlMessage extends Serializable
```

Comment:
We have always had this gotcha, I think it's documented. I dislike it and wanted to special case java.io.Serialization but someone (I think Roland) was against that. We have this open issue https://github.com/akka/akka/issues/17252